### PR TITLE
Update modules section

### PR DESCRIPTION
**Pull request by ignacy**
Update about new options and templates

### CODE REVIEW

File: marketplace_builder/views/pages/how-platformos-works/platformos-workflow/marketplace-kit.liquid [hunk 1]
```diff
@@ -167,6 +167,56 @@ Enables sync mode: it checks what's different between the server and the local e
...
+|---------  |-----------------------  |-----------------------  |
+| modules list | `marketplace-kit modules list <environment>`   | `marketplace-kit modules list staging`  |
+| modules remove | `marketplace-kit modules remove <environment> <name>` | `marketplace-kit modules remove staging admin_module` |
+
+##### module templates
+
+Templates provide automatic processing which will allows easier configuration of modules. For example it is possible upon installing a module to specify URIs under which pages will be available after the module is installed.
+This work both in sync and deploy mode.
+
+Markup is very simple ERB styke: `<%=` `=%>` there is no logic supported, only available filter is `&` which will unescape the value provided by the user (by default they are all escaped).
```
Comment (pavelloz) [inline marketplace_builder/views/pages/how-platformos-works/platformos-workflow/marketplace-kit.liquid]:
```suggestion
Markup is commonly used ERB/EJS stye: `<%=` `=%>` there is no logic supported, only available filter is `&` which will unescape the value provided by the user (by default they are all escaped).
```

File: marketplace_builder/views/pages/how-platformos-works/platformos-workflow/marketplace-kit.liquid [hunk 2]
```diff
@@ -167,6 +167,56 @@ Enables sync mode: it checks what's different between the server and the local e
...
+
+Values for variables have to be provided in toplevel `"template-values.json"` but the location of configuration file can be set using `TEMPLATE_VALUES_FILE_PATH` variable.
```
Comment (pavelloz) [inline marketplace_builder/views/pages/how-platformos-works/platformos-workflow/marketplace-kit.liquid]: Example use of `TEMPLATE_VALUES_FILE_PATH` would be nice

**Review by diana-lakatos — COMMENTED** [review]
Reviewed and edited, should be good to go after addressing Pawel's remaining feedback.

File: marketplace_builder/views/pages/how-platformos-works/platformos-workflow/marketplace-kit.liquid (hunk 1 above)
Comment (diana-lakatos) [reply marketplace_builder/views/pages/how-platformos-works/platformos-workflow/marketplace-kit.liquid]: Thank you, edited.

**Comment by pavelloz** [comment]
:)

**Pull request merged**